code/modules/uplink/uplink_items/job.dm

```diff
@@ -193,7 +193,15 @@
 	cost = 5 //you need two for full damage, so total of 10 for maximum damage
 	limited_stock = 2 //you can't use more than two!
 	restricted_roles = list("Shaft Miner")
-
+	
+/datum/uplink_item/role_restricted/syndicate_trophy
+	name = "Syndicate Crusher Trophy"
+	desc = "A relic seized by the syndicate that enables the Proto Kinetic Crusher to mark humanoid targets instead of the gross beasts of lavaland"
```

Comment:
```suggestion
	desc = "A relic seized by the Syndicate that enables the proto-kinetic crusher to mark humanoid targets instead of the gross beasts of Lavaland."
```

Reply:
how the fuck did i miss these things i minored in english